Alloy/common/constants.js

```diff
@@ -204,7 +204,6 @@ exports.IMPLICIT_NAMESPACES = {
 	CoverFlowView: NS_TI_UI_IOS,
 	DocumentViewer: NS_TI_UI_IOS,
 	LivePhotoView: NS_TI_UI_IOS,
-	NavigationWindow: NS_TI_UI_IOS,
```

Comment:
Will the old <NavigationWindow> still work if SDK < 7.5.0 is targeted?

Reply:
Yup.